cmake/ctest/drivers/geminga/TrilinosCTestDriverCore.geminga.gcc-cuda.cmake

```diff
@@ -71,6 +71,9 @@ MACRO(TRILINOS_SYSTEM_SPECIFIC_CTEST_DRIVER)
     SET(BUILD_DIR_NAME ${COMM_TYPE}-${BUILD_TYPE}_${BUILD_NAME_DETAILS})
   ENDIF()
 
+  SET(Trilinos_REPOSITORY_LOCATION_DEFAULT "github.com:trilinos/Trilinos.git")
+  SET(Trilinos_REPOSITORY_LOCATION_NIGHTLY_DEFAULT "github.com:trilinos/Trilinos.git")
```

Comment:
Shouldn't this be `muelu/Trilinos.git`?

Reply:
This is the default:
https://github.com/trilinos/Trilinos/blob/94ff67265ef757c28bcd31448296c5372e991e27/cmake/ctest/TrilinosCTestDriverCore.cmake#L15-L16
and to clone Trilinos, I use `git@github.com:trilinos/Trilinos.git`.
So I figured that this might be the correct format.

Reply:
Maybe we need the `git@`?

Reply:
> This is the default:
> 
> https://github.com/trilinos/Trilinos/blob/94ff67265ef757c28bcd31448296c5372e991e27/cmake/ctest/TrilinosCTestDriverCore.cmake#L15-L16
> 
> and to clone Trilinos, I use `git@github.com:trilinos/Trilinos.git`.
> So I figured that this might be the correct format.

What I really was asking was whether you intended to pull from the repo at the trilinos project, or from the repo at the MueLu project.

Reply:
Anyway, this can be corrected later.  I'll approve and let's see what happens.

Reply:
D'oh!